.spi.yml

```diff
@@ -1,20 +1,5 @@
 version: 1
 builder:
   configs:
-  - platform: macosXcodebuild
-    target: AWSSTS
```

Comment:
`macosXcodebuild` doesn't support the `target` flag, it needs `scheme` instead. (`--target` is a `swift build` parameter, `--scheme` an `xcodebuild` parameter. We should perhaps have collapsed them into one but it wasn't clear at the time if they'd be used synonymously.)